src/main/java/io/sc3/plethora/gameplay/modules/keyboard/KeyboardModuleItem.kt

```diff
@@ -118,7 +118,10 @@ class KeyboardModuleItem(settings: Settings) : ModuleItem("keyboard", settings)
       // BlockEntityHelpers.isUsable: the base usable check
       // AbstractComputerBlockEntity: lootable locked check, BEH.isUsable
       // CommandComputerBlockEntity:  isCommandUsable, AbstractComputerBlockEntity.isUsable
-      if (blockEntity is CommandComputerBlockEntity && !CommandComputerBlockEntity.isCommandUsable(player)) {
+
+
+
+      if (blockEntity.family == ComputerFamily.COMMAND && !blockEntity.serverComputer?.level?.server?.areCommandBlocksEnabled()!!) {
```

Comment:
I think we can use `checkUsable` here?

```suggestion
      if (!blockEntity.family.checkUsable(player))`
```

Reply:
Yeah that seems simpler to understand as well, I'll make the change and test it out before committing.
Thanks for the review.

Reply:
I've updated this and experienced no problems with they keyboard in my world.